### PR TITLE
DOCSP-13866 run a command usage example

### PR DESCRIPTION
**Pull request by kyuan-mongodb**
## Pull Request Info

Added Usage Examples > Run a Command

### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-13866

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/golang/docsworker-xlarge/DOCSP-13866-CommandUsageExample/usage-examples/command/

### Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Does it render on staging correctly?
- [x] Are all the links working?
- [x] Are the staging links in the PR description updated?

### CODE REVIEW

File: source/usage-examples/command.txt [hunk 1]
```diff
@@ -4,16 +4,52 @@ Run a Command
...
 .. default-domain:: mongodb
 
-You can run all raw database operations using the
-``MongoDatabase.runCommand()`` method. A raw database operation is a
-command you can execute directly on the MongoDB server CLI. These
-commands include administrative and diagnostic tasks, such as fetching
-server stats or initializing a replica set. Call the ``runCommand()``
-method with a Bson command object on an instance of a ``MongoDatabase``
-to run your raw database operation.
+You can run commands directly on the MongoDB server CLI using the
```
Comment (biniona-mongodb) [inline source/usage-examples/command.txt:7]:
```suggestion
You can run commands directly on your MongoDB server using the
```

Question (possible Issue): This sounds odd, as "CLI" implies a Command Line, and I don't think this is executed from a command line (I think the server just runs the command). I see this is also the phrasing in the Java Docs, however it does not seem to be the phrasing in the Node Docs or the [Mongosh docs](https://docs.mongodb.com/manual/reference/method/db.runCommand/). Can this be double checked?

File: source/usage-examples/command.txt [hunk 2]
```diff
@@ -4,16 +4,52 @@ Run a Command
...
+``RunCommand()`` method. 
+
+The following example passes a document containing the ``"dbStats"``
+command to the ``RunCommand()`` method, which requests for statistics
```
Comment (biniona-mongodb) [inline source/usage-examples/command.txt:11]:
```suggestion
command to the ``RunCommand()`` method, which retrieves statistics
```
Suggestion: "Requests for" sounds odd to me. Does "retrieves" fit this better?

File: source/usage-examples/command.txt [hunk 3]
```diff
@@ -4,16 +4,52 @@ Run a Command
...
+   :language: go
+   :dedent:
+
+Click `here <{+example+}/command.go>`__ to see a fully runnable example.
+
+Expected Result
+---------------
+
+After running the preceding code snippet, the ``result`` variable should
+contain the following:
```
Comment (biniona-mongodb) [inline source/usage-examples/command.txt:29]:
```suggestion
contain the following cursor (maybe struct?): 
```
Suggestion: Could this be more specific? I am not sure what I am looking at, and needed to consult API docs to try and figure it out.

File: source/usage-examples/command.txt [hunk 4]
```diff
@@ -4,16 +4,52 @@ Run a Command
...
+      {dataSize 14014074} {storageSize 8257536} 
+      ... ]
 
 .. note::
-    Use the :manual:`mongo shell </reference/mongo-shell>` for
-    administrative tasks instead of the Go driver whenever possible,
-    since these tasks are often quicker and easier to implement with the
-    shell than in a Go application.
+
+    The ``result`` variable may vary depending on your data set.
```
Comment (biniona-mongodb) [inline source/usage-examples/command.txt:44]:
```suggestion
    The ``result`` variable may vary depending the contents of your collection.
```
Nit: This struck me as odd as the data set we are providing is in theory the same for everyone. I think this has more to do with the readers "collection" (specific of what is on their machine right now) than their "data set" (which in my head is the thing they downloaded from atlas the first time).

**Review by biniona-mongodb — CHANGES_REQUESTED** [review]
The things that made me request changes were the phrase "CLI" and that the type of the result was not listed, other wise would have approved. Looks great! After those are addressed/discussed down to approve!

**Review by biniona-mongodb — APPROVED** [review]
LGTM

**Comment by kyuan-mongodb** [comment]
Hi @kevinAlbs, This PR is ready for review.

**Review by benjirewis — APPROVED** [review]
LGTM!

**Pull request merged**